docs/operations/memory_hotplug.md

```diff
@@ -65,7 +69,7 @@ The VM-level configuration will take precedence over the cluster-wide one.
 
 ## Memory Hotplug in Action
 
-First we enable the `VMLiveUpdateFeatures` feature gate and set `LiveMigrate` as `workloadUpdateStrategy` in the KubeVirt CR.
+First we enable the `VMLiveUpdateFeatures` feature gate, set the rollout strategy to `LiveUpdate` and set `LiveMigrate` as `workloadUpdateStrategy` in the KubeVirt CR.
```

Comment:
nit: not sure whether we want to include the rollout strategy update again below, just for completeness? (I've seen that it's been described above also, but maybe the reader just reads and follows this section)

Or at least add a "see above"